core/testdata/tomlspecs/vrf-v2-spec.toml

```diff
@@ -19,10 +19,10 @@ encode_tx    [type=ethabiencode
               abi="fulfillRandomWords(bytes proof)"
               data="{\\"proof\\": $(vrf.proof)}"]
 estimate_gas [type=estimategaslimit
-			  to="%s"
```

Comment:
Does removing these format specifiers not give errors?

Reply:
its just to help QA folks/anyone when copy pasting this spec. It should have been internally consistent before i.e. had the example address specified everywhere. Copy paste bug from testdata/testspecs